Sources/Purchasing/Configuration.swift

```diff
@@ -164,6 +167,12 @@ import Foundation
             return self
         }
 
+        /// Set `platformInfo`. Internal details, no need to expose.
+        func with(platformInfo: Purchases.PlatformInfo) -> Builder {
```

Comment:
Maybe make this explicitly `internal` instead of putting that in the comment?

Reply:
Fair